README.md

```diff
@@ -99,68 +117,72 @@ Biometry Exception Types:
 
 If it's a cancellation error, `OperationCanceledException` is thrown.
 
-### GetGapabilites
+#### GetGapabilites
```

Comment:
Instead of having everything under the "Methods" header, we could have all features at the same level. Instead of "Methods" this section could be about error handling because that's what is seems to be about.